plugins/nordic-hid/README.md

```diff
@@ -45,6 +60,13 @@ This plugin uses the following plugin-specific quirks:
 Explicitly set the expected bootloader type: "B0" or "MCUBOOT"
 This quirk must be set for devices without support of `bootloader variant` DFU option.
 
+### NordicHidGeneration
+
+Explicitly set the expected generation.
+Only "default" generation can be set by the quirk file.
+Other value can only be provided by device.
+This quirk must be set for devices without support of `devinfo` DFU option.
```

Comment:
```suggestion
This quirk must be set for devices without a support of the `devinfo` DFU option.
```
Or, would it be correct to say: (...) for devices that do not support the `devinfo` DFU option. (?) 
If so, this version would be preferable

Reply:
Should be ok, aligned